.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- HTML Comments can be left in place or removed. -->
+
+#### Proposed Changes ####
+
+<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. -->
+
+<!-- Does this change require an update to documentation? -->
+
+#### Types of Changes ####
+
+<!-- What types of changes does your code introduce to RKE2 or k3s? Bugfix, New Feature, Breaking Change, etc -->
+
+#### Testing ####
+
+<!-- Is this change covered by testing? If not, consider adding a Unit or Integration test. -->
+<!-- Answer the checklist bellow  -->
+
+Checklist:
+1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.
+
+
+2. Verify code lint; we should not have errors.
+
+
+3. Update the documentation if needed.
+
+
+4. Update makefile and docker run if adding new tests.
+
+
+5. Run your tests at least 4 times with all configurations needed and possible.
+
+
+6. If needed test with different os types.
+
+
+#### Linked Issues ####
+
+<!-- Link any related issues, pull-requests, or commit hashes that are relevant to this pull request. If you are opening a PR without a corresponding issue please consider creating one first, at https://github.com/rancher/distros-test-framework/issues . A functional example will greatly help QA with verifying/reproducing a bug or testing new features. -->
```

Comment:
update this comment to also include qa-tasks repo issues, and remove the last line: "A functional example will greatly help QA with verifying/reproducing a bug or testing new features."